Source/Base/ASAvailability.h

```diff
@@ -84,12 +84,28 @@
   #error "ASTEXTNODE_EXPERIMENT_GLOBAL_ENABLE is unavailable. See ASConfiguration.h."
 #endif
 
-#define AS_PIN_REMOTE_IMAGE __has_include(<PINRemoteImage/PINRemoteImage.h>)
-#define AS_IG_LIST_KIT __has_include(<IGListKit/IGListKit.h>)
-#define AS_IG_LIST_DIFF_KIT __has_include(<IGListDiffKit/IGListDiffKit.h>)
-
-/**
- * For IGListKit versions < 3.0, you have to use IGListCollectionView.
- * For 3.0 and later, that class is removed and you use UICollectionView.
- */
-#define IG_LIST_COLLECTION_VIEW __has_include(<IGListKit/IGListCollectionView.h>)
+#if !SWIFT_PACKAGE
+    #define AS_PIN_REMOTE_IMAGE __has_include(<PINRemoteImage/PINRemoteImage.h>)
+    #define AS_IG_LIST_KIT __has_include(<IGListKit/IGListKit.h>)
+    #define AS_IG_LIST_DIFF_KIT __has_include(<IGListDiffKit/IGListDiffKit.h>)
+
+    /**
+     * For IGListKit versions < 3.0, you have to use IGListCollectionView.
+     * For 3.0 and later, that class is removed and you use UICollectionView.
+     */
+    #define IG_LIST_COLLECTION_VIEW __has_include(<IGListKit/IGListCollectionView.h>)
+#else
+    #ifndef AS_PIN_REMOTE_IMAGE
+        #define AS_PIN_REMOTE_IMAGE 1
+    #endif
+
+    #ifndef AS_IG_LIST_KIT
```

Comment:
Was this intended? My project doesn't use IGListKit but does use ASPINRemoteImageDownloader which requires AS_PIN_REMOTE_IMAGE to be set.

Reply:
Also, I can't seem to be able to reference ASPINRemoteImageDownloader for some reason. I had to comment out the `#if AS_PIN_REMOTE_IMAGE` conditionals entirely in order for my project to reference it. Not sure if it's an SPM package configuration issue or something else.